content/docs/interface/example0/_index.md

```diff
@@ -574,18 +576,458 @@ finalise(handle)
+```
+
+#### Working *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/attach_issue.R /Users/SoniaM/datastore/coderun/20210511-231444/
+  latest_commit: 358f64c4044f3b3f761865ee8e9f4375cf41d155
+  remote_repo: https://github.com/fake_org/fake_repo
+read:
+- data_product: findme/test/array2b_5b1acbdd0b7a83ebaf5aba717a26ab0a4a081964
+  use:
+    version: 0.1.0
+write: []
+```
+
+#### Submission script (R)
+
+```R
+library(rFDP)
+
+# Initialise Code Run
+config <- file.path(Sys.getenv("FDP_CONFIG_DIR"), "config.yaml")
+script <- file.path(Sys.getenv("FDP_CONFIG_DIR"), "script.sh")
+handle <- initialise(config, script)
+
+issue <- "issue with script"
+severity <- 7
+
+raise_issue_script(handle = handle,
+                   issue = script_issue,
+                   severity = script_severity)
+
+finalise(handle)
+```
+
 ## Attach issue to GitHub repository
 
+#### User written *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/attach_issue.R /Users/SoniaM/datastore/coderun/20210511-231444/
+read:
+- data_product: findme/test/array2b_5b1acbdd0b7a83ebaf5aba717a26ab0a4a081964
+  version: 0.1.0
+```
+
+#### Working *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
+  local_repo: local_repo
+  script: |-
+    R -f simple_working_examples/attach_issue.R /Users/SoniaM/datastore/coderun/20210511-231444/
+  latest_commit: 6b23ec822bfd7ea5f419c70ce18fb73b59c90754
+  remote_repo: https://github.com/fake_org/fake_repo
+read:
+- data_product: findme/test/array2b_5b1acbdd0b7a83ebaf5aba717a26ab0a4a081964
+  use:
+    version: 0.1.0
+write: []
+```
+
+#### Submission script (R)
+
+```R
+library(rFDP)
+
+# Initialise Code Run
+config <- file.path(Sys.getenv("FDP_CONFIG_DIR"), "config.yaml")
+script <- file.path(Sys.getenv("FDP_CONFIG_DIR"), "script.sh")
+handle <- initialise(config, script)
+
+issue <- "issue with repo"
+severity <- 7
+
+raise_issue_repo(handle = handle,
+                 issue = repo_issue,
+                 severity = repo_severity)
+
+finalise(handle)
+```
+
 ## Attach issue to external object
 This is not something we want to do.
```

Comment:
It's a bit indirect, but if we want to do this, then we need to attach the issue to the `whole_object` component of the object associated with the data product associated with the external object. I assume it's possible to do this, because the data product is the alias for the external object anyway? If so, presumably we can add mention here it is just the same as attaching an issue to a whole data product?
```suggestion
Attaching an issue to an external object is just the same as attaching an issue to the data product that the external object is attached to.
```

Reply:
The question is... do we want to do this? – previously the answer was no, but I'm happy to implement if that's changed.

Reply:
What I mean is, we can already do this by attaching an issue to its data product. I may have phrased it badly though. However, I agree(?) that we still don't want to have any additional functionality to enable it...

Reply:
If we don’t want the functionality, I’d suggest leaving my comment as is to avoid confusion?

Reply:
My reasoning for adding this to the examples page was to put it in writing that this was not to be implemented.